lnst/RecipeCommon/Perf/Evaluators/BaselineCPUAverageEvaluator.py

```diff
@@ -77,28 +77,30 @@ def compare_result_with_baseline(
         result: PerfMeasurementResults,
         baseline: PerfMeasurementResults,
         result_index: int = 0
-    ) -> Tuple[ResultType, List[str]]:
+    ) -> List[MetricComparison]:
         comparison = ResultType.FAIL
-        text = []
 
-        host = result.host.hostid
-        metric = f"{result_index}_{host}"
+        metric_name = f"{result_index}_utilization"
```

Comment:
if we are going to rename metric, let's use something more descriptive. what about `{result_index}_{generator/receiver}_utilization`/`{result_index}_{generator/receiver}_utilization_reversed`

Reply:
The evaluator has no clue who is generator/receiver, so this is not possible. But we can still include the `{host}` in the string.

Reply:
to expand on what jan said, generator/receiver refers to **Flows** specifically, cpu measurements are completely independent of that, we can run cpu measurements even on nodes that are in between the generator and receiver.

i'm not 100% sure about including the host either, it's already part of the description of the evaluation:
```
    FAIL 191_TestResult:
        CPU Baseline average evaluation for Host host2:
        FAIL: CPU cpu: 191_utilization no threshold found
```

I want to keep this simple and only have the name of the metric in there to keep some sort of consistency across all different types of measurement evaluators which look at more/fewer metrics. We could for example add more metrics to the cpu evaluator and then the host id would be repeated everywhere...

So to summarize, IMO, it is actually very descriptive to use only `utilization`, as this is already `hostX` cpu measurement evaluation and we're looking at the `utilization` metric instead of for example `soft_irq`.

Reply:
I'm fine with keeping just the `{result_no}_utilization`.